<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Cdk7_paper_figure</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="README_files/libs/clipboard/clipboard.min.js"></script>
<script src="README_files/libs/quarto-html/quarto.js"></script>
<script src="README_files/libs/quarto-html/popper.min.js"></script>
<script src="README_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="README_files/libs/quarto-html/anchor.min.js"></script>
<link href="README_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="README_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="README_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="README_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="README_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Cdk7_paper_figure</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="percent-similarity" class="level3">
<h3 class="anchored" data-anchor-id="percent-similarity">Percent similarity</h3>
<div class="cell">
<div class="cell-output-display">
<p><img src="README_files/figure-html/percent_similar-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="enzyme-assay" class="level3">
<h3 class="anchored" data-anchor-id="enzyme-assay">enzyme assay</h3>
<section id="section" class="level4">
<h4 class="anchored" data-anchor-id="section"></h4>
</section>
<section id="enzyme-assay-pvalue-at-last-time-point" class="level4">
<h4 class="anchored" data-anchor-id="enzyme-assay-pvalue-at-last-time-point">Enzyme assay pvalue at last time-point</h4>
<div class="cell">
<div class="cell-output-display">
<p><img src="README_files/figure-html/pval_last-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
</section>
<section id="dose-response-curve-2-fold-serial-dilution-15102024" class="level3">
<h3 class="anchored" data-anchor-id="dose-response-curve-2-fold-serial-dilution-15102024">Dose response curve 2 fold serial dilution (15/10/2024)</h3>
<p>Experimental setup</p>
<ol type="1">
<li>Cells were grown overnight in YPD media</li>
<li>OD for O/N cells (OD=28.84) was determined and diluted to OD600 = 0.01/ml in 15ml (6.935µl) fresh YPD.</li>
<li>Cells were aliquoted in 1ml tubes (14x)</li>
<li>Mev stock (100mg/ml) and BS181 stock (40mg/ml) was serially diluted in DMSO and cells were treated with 1µl of Mev and 2.5µl of BS181 from each dilution and only DMSO (0µg/ml) as control</li>
<li>Cells were aliquoted in 96-well plate, 200µl/well.</li>
<li>Cell growth was monitored by recording OD600 every 20min up-to 40hrs with constant shaking at 800rpm at 30C using <a href="https://www.google.com/url?sa=t&amp;source=web&amp;rct=j&amp;opi=89978449&amp;url=https://www.agilent.com/en/product/microplate-instrumentation/microplate-readers/multiplate-absorbance-reader/biotek-logphase-600-microbiology-reader-1623185&amp;ved=2ahUKEwiTyenf3v2IAxWMh1YBHWNKFdMQFnoECBcQAQ&amp;usg=AOvVaw0jzPdCYQNt27zqyw6PwRos">Agilent Biotek logphase 600</a> plate reader</li>
<li>Cell growth is plotted as Time Vs OD600. Concentration is indicated in the brackets as µg/ml.</li>
</ol>
<section id="h99" class="level4">
<h4 class="anchored" data-anchor-id="h99">H99</h4>
<div class="cell">
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 146 Columns: 69
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
dbl  (68): Mev(100)_1, Mev(100)_2, Mev(100)_3, Mev(100)_4, BS181(100)_1, BS1...
time  (1): Time

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: The `.dots` argument of `group_by()` is deprecated as of dplyr 1.0.0.
ℹ The deprecated feature was likely used in the dplyr package.
  Please report the issue at &lt;https://github.com/tidyverse/dplyr/issues&gt;.</code></pre>
</div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/dose_curve_17Oct24-1.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Conclusions</p>
<ol type="1">
<li>For Mev, 12.5µg/ml is the inhibitory concentration</li>
<li>For BS181, 50µg/ml concentration shows complete inhibition</li>
<li>Dose response assay shows that stark but not gradual inhibition in growth at the described concentration</li>
</ol>
</section>
<section id="cak-tag" class="level4">
<h4 class="anchored" data-anchor-id="cak-tag">CAK-tag</h4>
<div class="cell">
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 84 Columns: 97
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
dbl  (96): KN99(100)_1, KN99(100)_2, KN99(100)_3, CM2444(100)_1, CM2444(100)...
time  (1): Time

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/dose_curve_20Dec24-1.png" class="img-fluid" width="672"></p>
</div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/dose_curve_20Dec24-2.png" class="img-fluid" width="672"></p>
</div>
</div>
<p>Conclusions</p>
<ol type="1">
<li>WT-KN99 and Cdk7 tagged strains have MIC of 12.5µg/ml for Mevociclib</li>
<li>The MIC values are same as Cn H99</li>
<li>Together results suggests
<ul>
<li><p>Irrespective of strain (KN99 or H99) C. neoformans have same MIC for Cdk7 inhibitors</p></li>
<li><p>Adding tags to the Cdk7-complex does not alter the growth inhibition in presence of Cdk7 inhibitors</p></li>
</ul></li>
</ol>
</section>
</section>
<section id="phospho-proteomics" class="level3">
<h3 class="anchored" data-anchor-id="phospho-proteomics">Phospho-proteomics</h3>
<section id="phospho-proteomics-1" class="level4">
<h4 class="anchored" data-anchor-id="phospho-proteomics-1">Phospho-proteomics</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>phospho_tb <span class="ot">&lt;-</span> readr<span class="sc">::</span><span class="fu">read_delim</span>(<span class="st">"data_files/Phospho_RawData.txt"</span>, <span class="at">delim=</span><span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">col_names =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 961 Columns: 9
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr (5): Accession, Sample_With_1, Sample_With_2, Sample_Without_1, Sample_W...
dbl (4): TotalPeptides, UniquePeptides, Abundances_With, Abundances_Without

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>phospho_tb <span class="ot">&lt;-</span> phospho_tb <span class="sc">%&gt;%</span> </span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(stringr<span class="sc">::</span><span class="fu">str_starts</span>(Accession, <span class="st">"CNAG"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">rowwise</span>() <span class="sc">%&gt;%</span> <span class="co"># remove rows with no abundance reported</span></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(<span class="sc">!</span><span class="fu">all</span>(<span class="fu">is.na</span>(dplyr<span class="sc">::</span><span class="fu">c_across</span>(<span class="fu">c</span>(Abundances_With, Abundances_Without)))))</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a><span class="co"># remove rows having peak "Not found" in both the replicate</span></span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>phospho_tb_mev <span class="ot">&lt;-</span> phospho_tb <span class="sc">%&gt;%</span></span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="fu">c</span>(Accession, Sample_With_1,Sample_With_2)) <span class="sc">%&gt;%</span></span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">rowwise</span>() <span class="sc">%&gt;%</span></span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">filter</span>(<span class="fu">sum</span>(dplyr<span class="sc">::</span><span class="fu">c_across</span>(<span class="dv">2</span><span class="sc">:</span><span class="dv">3</span>) <span class="sc">!=</span> <span class="st">"Not Found"</span>) <span class="sc">==</span> <span class="dv">2</span>) <span class="sc">%&gt;%</span></span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">ungroup</span>()</span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a>phospho_tb_untreated <span class="ot">&lt;-</span> phospho_tb <span class="sc">%&gt;%</span></span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="fu">c</span>(Accession, Sample_Without_1,Sample_Without_2)) <span class="sc">%&gt;%</span></span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">rowwise</span>() <span class="sc">%&gt;%</span></span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">filter</span>(<span class="fu">sum</span>(dplyr<span class="sc">::</span><span class="fu">c_across</span>(<span class="dv">2</span><span class="sc">:</span><span class="dv">3</span>) <span class="sc">!=</span> <span class="st">"Not Found"</span>) <span class="sc">==</span> <span class="dv">2</span>) <span class="sc">%&gt;%</span></span>
<span id="cb6-17"><a href="#cb6-17" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">ungroup</span>()</span>
<span id="cb6-18"><a href="#cb6-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-19"><a href="#cb6-19" aria-hidden="true" tabindex="-1"></a><span class="co"># create a combined list of mev and untreated</span></span>
<span id="cb6-20"><a href="#cb6-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-21"><a href="#cb6-21" aria-hidden="true" tabindex="-1"></a>phospho_combined_list <span class="ot">&lt;-</span> dplyr<span class="sc">::</span><span class="fu">bind_rows</span>(</span>
<span id="cb6-22"><a href="#cb6-22" aria-hidden="true" tabindex="-1"></a>                            phospho_tb_untreated <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">select</span>(Accession),</span>
<span id="cb6-23"><a href="#cb6-23" aria-hidden="true" tabindex="-1"></a>                           phospho_tb_mev <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">select</span>(Accession)</span>
<span id="cb6-24"><a href="#cb6-24" aria-hidden="true" tabindex="-1"></a>                          ) <span class="sc">%&gt;%</span></span>
<span id="cb6-25"><a href="#cb6-25" aria-hidden="true" tabindex="-1"></a>                          dplyr<span class="sc">::</span><span class="fu">distinct</span>()</span>
<span id="cb6-26"><a href="#cb6-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-27"><a href="#cb6-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-28"><a href="#cb6-28" aria-hidden="true" tabindex="-1"></a>phospho_filtered <span class="ot">&lt;-</span> phospho_tb <span class="sc">%&gt;%</span></span>
<span id="cb6-29"><a href="#cb6-29" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">filter</span>(Accession <span class="sc">%in%</span> phospho_combined_list<span class="sc">$</span>Accession) <span class="sc">%&gt;%</span></span>
<span id="cb6-30"><a href="#cb6-30" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">LFC =</span> <span class="fu">log2</span>(Abundances_With<span class="sc">/</span>Abundances_Without),</span>
<span id="cb6-31"><a href="#cb6-31" aria-hidden="true" tabindex="-1"></a>                                <span class="at">class=</span>dplyr<span class="sc">::</span><span class="fu">if_else</span>(LFC<span class="sc">&gt;</span><span class="dv">1</span>, <span class="st">"Up"</span>,</span>
<span id="cb6-32"><a href="#cb6-32" aria-hidden="true" tabindex="-1"></a>                                      dplyr<span class="sc">::</span><span class="fu">if_else</span>(LFC<span class="sc">&lt;</span> <span class="sc">-</span><span class="dv">1</span>, <span class="st">"Down"</span>, <span class="st">"No-change"</span>))) <span class="sc">%&gt;%</span></span>
<span id="cb6-33"><a href="#cb6-33" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">class=</span>dplyr<span class="sc">::</span><span class="fu">if_else</span>(<span class="fu">is.na</span>(Abundances_Without), <span class="st">"Mev_Only"</span>, dplyr<span class="sc">::</span><span class="fu">if_else</span>(<span class="fu">is.na</span>(Abundances_With), <span class="st">"WT_Only"</span>, class)))</span>
<span id="cb6-34"><a href="#cb6-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-35"><a href="#cb6-35" aria-hidden="true" tabindex="-1"></a>Phospho_stats <span class="ot">&lt;-</span> phospho_filtered <span class="sc">%&gt;%</span> </span>
<span id="cb6-36"><a href="#cb6-36" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">count</span>(class) </span>
<span id="cb6-37"><a href="#cb6-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-38"><a href="#cb6-38" aria-hidden="true" tabindex="-1"></a>Phospho_stats <span class="ot">&lt;-</span> tibble<span class="sc">::</span><span class="fu">tribble</span>(</span>
<span id="cb6-39"><a href="#cb6-39" aria-hidden="true" tabindex="-1"></a>                        <span class="sc">~</span>class,   <span class="sc">~</span>n,</span>
<span id="cb6-40"><a href="#cb6-40" aria-hidden="true" tabindex="-1"></a>                          <span class="st">"Up"</span>, 134L,</span>
<span id="cb6-41"><a href="#cb6-41" aria-hidden="true" tabindex="-1"></a>                   <span class="st">"No-change"</span>, 466L,</span>
<span id="cb6-42"><a href="#cb6-42" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"Down"</span>, 189L,</span>
<span id="cb6-43"><a href="#cb6-43" aria-hidden="true" tabindex="-1"></a>                    <span class="st">"Mev_Only"</span>,  40L,</span>
<span id="cb6-44"><a href="#cb6-44" aria-hidden="true" tabindex="-1"></a>                     <span class="st">"WT_Only"</span>,  14L</span>
<span id="cb6-45"><a href="#cb6-45" aria-hidden="true" tabindex="-1"></a>                   )</span>
<span id="cb6-46"><a href="#cb6-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-47"><a href="#cb6-47" aria-hidden="true" tabindex="-1"></a>Phospho_stats <span class="sc">%&gt;%</span> </span>
<span id="cb6-48"><a href="#cb6-48" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">class=</span>forcats<span class="sc">::</span><span class="fu">as_factor</span>(class)) <span class="sc">%&gt;%</span></span>
<span id="cb6-49"><a href="#cb6-49" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">ggplot</span>(ggplot2<span class="sc">::</span><span class="fu">aes</span>(class, n, <span class="at">label=</span>n, <span class="at">fill=</span>class))<span class="sc">+</span></span>
<span id="cb6-50"><a href="#cb6-50" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">geom_col</span>(<span class="at">alpha=</span><span class="fl">0.8</span>, <span class="at">color=</span><span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb6-51"><a href="#cb6-51" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">theme_bw</span>()<span class="sc">+</span></span>
<span id="cb6-52"><a href="#cb6-52" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">geom_text</span>(<span class="at">vjust=</span><span class="sc">-</span><span class="dv">1</span>)<span class="sc">+</span></span>
<span id="cb6-53"><a href="#cb6-53" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">scale_fill_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">"red"</span>, <span class="st">"gray"</span>, <span class="st">"blue"</span>, <span class="st">"salmon"</span>, <span class="st">"cyan4"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/filter_phosphoData-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="co"># combine WT_only to Down and Mev_Only to Up</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>Phospho_stats_2 <span class="ot">&lt;-</span> phospho_filtered <span class="sc">%&gt;%</span> </span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">class2=</span>dplyr<span class="sc">::</span><span class="fu">if_else</span>(class<span class="sc">==</span><span class="st">"WT_Only"</span>, <span class="st">"Down"</span>, dplyr<span class="sc">::</span><span class="fu">if_else</span>(class<span class="sc">==</span><span class="st">"Mev_Only"</span>, <span class="st">"Up"</span>, class))) <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">count</span>(class2)</span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>Phospho_stats_2 <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>            dplyr<span class="sc">::</span><span class="fu">arrange</span>(<span class="fu">desc</span>(class2)) <span class="sc">%&gt;%</span> </span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>            dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">class2=</span>forcats<span class="sc">::</span><span class="fu">as_factor</span>(class2)) <span class="sc">%&gt;%</span></span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">ggplot</span>(ggplot2<span class="sc">::</span><span class="fu">aes</span>(class2, n, <span class="at">label=</span>n, <span class="at">fill=</span>class2))<span class="sc">+</span></span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">geom_col</span>(<span class="at">alpha=</span><span class="fl">0.8</span>, <span class="at">color=</span><span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">theme_bw</span>()<span class="sc">+</span></span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">geom_text</span>(<span class="at">vjust=</span><span class="sc">-</span><span class="dv">1</span>)<span class="sc">+</span></span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">scale_fill_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">"red"</span>, <span class="st">"gray"</span>, <span class="st">"blue"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/filter_phosphoData-2.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="lcms" class="level4">
<h4 class="anchored" data-anchor-id="lcms">LCMS</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>lcms_dat <span class="ot">=</span> readr<span class="sc">::</span><span class="fu">read_delim</span>(<span class="st">"data_files/LCMS_RawData.txt"</span>, <span class="at">delim=</span><span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 3247 Columns: 10
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr (7): Accession, Sample_With_1, Sample_With_2, Sample_With_3, Sample_With...
dbl (3): Unique Peptides, Abundances_With, Abundances_Without

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="co"># filter-out rows having "Not found" in atleast 2 of the three replicates</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>lcms_dat_Mev <span class="ot">&lt;-</span> lcms_dat <span class="sc">%&gt;%</span></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="fu">c</span>(Accession, Sample_With_1,Sample_With_2,Sample_With_3)) <span class="sc">%&gt;%</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">rowwise</span>() <span class="sc">%&gt;%</span></span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">filter</span>(<span class="fu">sum</span>(dplyr<span class="sc">::</span><span class="fu">c_across</span>(<span class="dv">2</span><span class="sc">:</span><span class="dv">4</span>) <span class="sc">!=</span> <span class="st">"Not Found"</span>) <span class="sc">&gt;=</span> <span class="dv">2</span>) <span class="sc">%&gt;%</span></span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">ungroup</span>()</span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>lcms_dat_WT <span class="ot">&lt;-</span> lcms_dat <span class="sc">%&gt;%</span></span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="fu">c</span>(Accession, Sample_Without_1,Sample_Without_2,Sample_Without_3)) <span class="sc">%&gt;%</span></span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">rowwise</span>() <span class="sc">%&gt;%</span></span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(<span class="fu">sum</span>(dplyr<span class="sc">::</span><span class="fu">c_across</span>(<span class="dv">2</span><span class="sc">:</span><span class="dv">4</span>) <span class="sc">!=</span> <span class="st">"Not Found"</span>) <span class="sc">&gt;=</span> <span class="dv">2</span>) <span class="sc">%&gt;%</span></span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">ungroup</span>()</span>
<span id="cb10-14"><a href="#cb10-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-15"><a href="#cb10-15" aria-hidden="true" tabindex="-1"></a>combined_genelist <span class="ot">&lt;-</span>  dplyr<span class="sc">::</span><span class="fu">bind_rows</span>(</span>
<span id="cb10-16"><a href="#cb10-16" aria-hidden="true" tabindex="-1"></a>                            lcms_dat_WT <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">select</span>(Accession),</span>
<span id="cb10-17"><a href="#cb10-17" aria-hidden="true" tabindex="-1"></a>                            lcms_dat_Mev <span class="sc">%&gt;%</span> dplyr<span class="sc">::</span><span class="fu">select</span>(Accession)</span>
<span id="cb10-18"><a href="#cb10-18" aria-hidden="true" tabindex="-1"></a>                          ) <span class="sc">%&gt;%</span></span>
<span id="cb10-19"><a href="#cb10-19" aria-hidden="true" tabindex="-1"></a>                            dplyr<span class="sc">::</span><span class="fu">distinct</span>()</span>
<span id="cb10-20"><a href="#cb10-20" aria-hidden="true" tabindex="-1"></a>lcms_filtered <span class="ot">&lt;-</span> lcms_dat <span class="sc">%&gt;%</span></span>
<span id="cb10-21"><a href="#cb10-21" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">filter</span>(Accession <span class="sc">%in%</span> combined_genelist<span class="sc">$</span>Accession) <span class="sc">%&gt;%</span></span>
<span id="cb10-22"><a href="#cb10-22" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">rowwise</span>() <span class="sc">%&gt;%</span></span>
<span id="cb10-23"><a href="#cb10-23" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">filter</span>(<span class="sc">!</span><span class="fu">any</span>(<span class="fu">is.na</span>(dplyr<span class="sc">::</span><span class="fu">c_across</span>(<span class="fu">c</span>(Abundances_With, Abundances_Without))))) <span class="sc">%&gt;%</span></span>
<span id="cb10-24"><a href="#cb10-24" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">filter</span>(stringr<span class="sc">::</span><span class="fu">str_starts</span>(Accession, <span class="st">"CNAG"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb10-25"><a href="#cb10-25" aria-hidden="true" tabindex="-1"></a>                  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">LFC =</span> <span class="fu">log2</span>(Abundances_With<span class="sc">/</span>Abundances_Without),</span>
<span id="cb10-26"><a href="#cb10-26" aria-hidden="true" tabindex="-1"></a>                                <span class="at">class=</span>dplyr<span class="sc">::</span><span class="fu">if_else</span>(LFC<span class="sc">&gt;</span><span class="dv">1</span>, <span class="st">"Up"</span>,</span>
<span id="cb10-27"><a href="#cb10-27" aria-hidden="true" tabindex="-1"></a>                                                     dplyr<span class="sc">::</span><span class="fu">if_else</span>(LFC<span class="sc">&lt;</span> <span class="sc">-</span><span class="dv">1</span>, <span class="st">"Down"</span>, <span class="st">"No-change"</span>)))</span>
<span id="cb10-28"><a href="#cb10-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-29"><a href="#cb10-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-30"><a href="#cb10-30" aria-hidden="true" tabindex="-1"></a>LCMS_stats <span class="ot">&lt;-</span> lcms_filtered <span class="sc">%&gt;%</span> </span>
<span id="cb10-31"><a href="#cb10-31" aria-hidden="true" tabindex="-1"></a>                    dplyr<span class="sc">::</span><span class="fu">count</span>(class) <span class="sc">%&gt;%</span></span>
<span id="cb10-32"><a href="#cb10-32" aria-hidden="true" tabindex="-1"></a>                    dplyr<span class="sc">::</span><span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb10-33"><a href="#cb10-33" aria-hidden="true" tabindex="-1"></a>                    dplyr<span class="sc">::</span><span class="fu">arrange</span>(<span class="fu">desc</span>(class)) <span class="sc">%&gt;%</span> </span>
<span id="cb10-34"><a href="#cb10-34" aria-hidden="true" tabindex="-1"></a>                    dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">class=</span>forcats<span class="sc">::</span><span class="fu">as_factor</span>(class))</span>
<span id="cb10-35"><a href="#cb10-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-36"><a href="#cb10-36" aria-hidden="true" tabindex="-1"></a>LCMS_stats <span class="sc">%&gt;%</span> ggplot2<span class="sc">::</span><span class="fu">ggplot</span>(ggplot2<span class="sc">::</span><span class="fu">aes</span>(class, n, <span class="at">label=</span>n, <span class="at">fill=</span>class))<span class="sc">+</span></span>
<span id="cb10-37"><a href="#cb10-37" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">geom_col</span>(<span class="at">alpha=</span><span class="fl">0.8</span>, <span class="at">color=</span><span class="st">"black"</span>) <span class="sc">+</span></span>
<span id="cb10-38"><a href="#cb10-38" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">theme_bw</span>()<span class="sc">+</span></span>
<span id="cb10-39"><a href="#cb10-39" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">geom_text</span>(<span class="at">vjust=</span><span class="sc">-</span><span class="dv">1</span>)<span class="sc">+</span></span>
<span id="cb10-40"><a href="#cb10-40" aria-hidden="true" tabindex="-1"></a>            ggplot2<span class="sc">::</span><span class="fu">scale_fill_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">"red"</span>, <span class="st">"gray"</span>, <span class="st">"blue"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/filter_lcms-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="compare-lcms-and-phospho-proteomics" class="level4">
<h4 class="anchored" data-anchor-id="compare-lcms-and-phospho-proteomics">Compare LCMS and Phospho-proteomics</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>dat_venn1 <span class="ot">&lt;-</span> readr<span class="sc">::</span><span class="fu">read_delim</span>(<span class="st">"data_files/LCSM_Phosph_VennInput.txt"</span>, <span class="at">delim=</span><span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">col_names =</span><span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: One or more parsing issues, call `problems()` on your data frame for details,
e.g.:
  dat &lt;- vroom(...)
  problems(dat)</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 2601 Columns: 4
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr (4): No-change_LCMS, No-change_Pho, Down_Pho, Up_Pho

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>gg_1 <span class="ot">=</span> <span class="fu">as.list</span>(dat_venn1)</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>v1 <span class="ot">=</span> <span class="fu">lapply</span>(gg_1,unique)</span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a>clean_list <span class="ot">&lt;-</span> <span class="fu">lapply</span>(v1, <span class="cf">function</span>(x) x[<span class="sc">!</span><span class="fu">is.na</span>(x)])</span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-9"><a href="#cb14-9" aria-hidden="true" tabindex="-1"></a><span class="fu">lengths</span>(clean_list)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>No-change_LCMS  No-change_Pho       Down_Pho         Up_Pho 
          2601            466            203            174 </code></pre>
</div>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(clean_list)<span class="ot">=</span><span class="fu">factor</span>(<span class="fu">names</span>(clean_list))</span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>upset_venn1 <span class="ot">&lt;-</span> UpSetR<span class="sc">::</span><span class="fu">upset</span>(UpSetR<span class="sc">::</span><span class="fu">fromList</span>(clean_list), </span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>          <span class="at">nsets =</span> <span class="fu">ncol</span>(dat_venn),</span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>          <span class="co">#order.by = c("degree"),</span></span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a>          <span class="at">keep.order=</span><span class="cn">TRUE</span>,<span class="at">sets=</span><span class="fu">names</span>(clean_list),</span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a>          <span class="at">sets.bar.color =</span> <span class="fu">c</span>( <span class="st">"grey"</span>,<span class="st">"blue"</span>, <span class="st">"red"</span>, <span class="st">"red"</span>),</span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>          <span class="at">text.scale=</span><span class="dv">2</span>, <span class="at">sets.x.label =</span> <span class="st">"No. of genes"</span>)</span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(upset_venn1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/lcms_phos_venn-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Convert matrix to tidy format</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>UpSetR<span class="sc">::</span><span class="fu">fromList</span>(clean_list) <span class="sc">%&gt;%</span></span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>        dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">gene =</span> <span class="fu">rownames</span>(.)) <span class="sc">%&gt;%</span></span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>        tidyr<span class="sc">::</span><span class="fu">pivot_longer</span>(<span class="sc">-</span>gene, <span class="at">names_to =</span> <span class="st">"set"</span>, <span class="at">values_to =</span> <span class="st">"present"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">filter</span>(present <span class="sc">==</span> <span class="dv">1</span>) <span class="sc">%&gt;%</span></span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">group_by</span>(gene) <span class="sc">%&gt;%</span></span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">summarise</span>(<span class="at">sets =</span> <span class="fu">paste</span>(<span class="fu">sort</span>(set), <span class="at">collapse =</span> <span class="st">","</span>)) <span class="ot">-&gt;</span> gene_intersections</span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a><span class="co"># See genes per intersection</span></span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a><span class="fu">table</span>(gene_intersections<span class="sc">$</span>sets)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
                    Down_Pho      Down_Pho,No-change_LCMS 
                          81                          122 
              No-change_LCMS No-change_LCMS,No-change_Pho 
                        2139                          254 
       No-change_LCMS,Up_Pho                No-change_Pho 
                          86                          212 
                      Up_Pho 
                          88 </code></pre>
</div>
</div>
</section>
<section id="growth-in-liquid-culture" class="level4">
<h4 class="anchored" data-anchor-id="growth-in-liquid-culture">Growth in liquid culture</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>growth_tube_dt <span class="ot">&lt;-</span> tibble<span class="sc">::</span><span class="fu">tribble</span>(</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>                    <span class="sc">~</span>Time,       <span class="sc">~</span>DMSO,     <span class="sc">~</span>SY1365,</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>                     <span class="st">"1h"</span>, <span class="fl">140.7514451</span>, <span class="fl">130.0578035</span>,</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>                     <span class="st">"2h"</span>, <span class="fl">183.1213873</span>, <span class="fl">153.0057803</span>,</span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>                     <span class="st">"3h"</span>, <span class="fl">228.9017341</span>, <span class="fl">176.2427746</span>,</span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>                     <span class="st">"4h"</span>, <span class="fl">279.2196532</span>,  <span class="fl">211.734104</span>,</span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a>                     <span class="st">"5h"</span>, <span class="fl">357.8034682</span>, <span class="fl">243.3526012</span>,</span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a>                    )</span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-10"><a href="#cb19-10" aria-hidden="true" tabindex="-1"></a>growth_tube_dt <span class="sc">%&gt;%</span> </span>
<span id="cb19-11"><a href="#cb19-11" aria-hidden="true" tabindex="-1"></a>  tidyr<span class="sc">::</span><span class="fu">gather</span>(Sample, Percentage, <span class="sc">-</span>Time) <span class="sc">%&gt;%</span></span>
<span id="cb19-12"><a href="#cb19-12" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(<span class="at">Sample=</span>forcats<span class="sc">::</span><span class="fu">as_factor</span>(Sample)) <span class="sc">%&gt;%</span></span>
<span id="cb19-13"><a href="#cb19-13" aria-hidden="true" tabindex="-1"></a>  ggplot2<span class="sc">::</span><span class="fu">ggplot</span>(ggplot2<span class="sc">::</span><span class="fu">aes</span>(Time, Percentage, <span class="at">fill=</span>Sample))<span class="sc">+</span></span>
<span id="cb19-14"><a href="#cb19-14" aria-hidden="true" tabindex="-1"></a>  ggplot2<span class="sc">::</span><span class="fu">geom_col</span>(<span class="at">color=</span><span class="st">"black"</span>)<span class="sc">+</span></span>
<span id="cb19-15"><a href="#cb19-15" aria-hidden="true" tabindex="-1"></a>  ggplot2<span class="sc">::</span><span class="fu">theme_bw</span>()<span class="sc">+</span></span>
<span id="cb19-16"><a href="#cb19-16" aria-hidden="true" tabindex="-1"></a>  ggplot2<span class="sc">::</span><span class="fu">facet_wrap</span>(<span class="sc">~</span>Sample, <span class="at">scales=</span><span class="st">"fixed"</span>)<span class="sc">+</span></span>
<span id="cb19-17"><a href="#cb19-17" aria-hidden="true" tabindex="-1"></a>  ggplot2<span class="sc">::</span><span class="fu">scale_fill_manual</span>(<span class="at">values=</span><span class="fu">c</span>(<span class="st">"grey"</span>, <span class="st">"darkgreen"</span>,<span class="st">"darkblue"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="README_files/figure-html/growth_in_tube-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>